test/extended/networking/egress_firewall.go

```diff
@@ -49,6 +56,25 @@ var _ = g.Describe("[sig-network][Feature:EgressFirewall]", func() {
 			})
 		},
 	)
+
+	// When OVNKubernetes subnet and coredns-ocp-dnsnameresolver plugins are enabled.
+	// coredns-ocp-dnsnameresolver plugin is a TechPreview in 4.16.
```

Comment:
Better not to mention the OCP release.
```suggestion
	// coredns-ocp-dnsnameresolver plugin is a TechPreview feature.
```